frontend/src/components/signup.js

```diff
@@ -1,10 +1,51 @@
 import React from 'react';
 
-const SignUp = () => (
-    <div>
+const SignUp = () => {
+    const handleSubmit = (evt) => {
+        evt.preventDefault();
+        console.log("Form submitted");
+    }
 
-    </div>
-);
+    return (
+            <div>
+            <form method="post" action="/st21-what-the-wine/public/index.html" onSubmit={handleSubmit}>
+            <fieldset>
+            <legend>Sign Up for personalized Wine Recommendations</legend>
+                <p>
+                    <label name="firstname">First Name</label>
+                    <input type="text" name="firstname"></input>
```

Comment:
One accessibility detail about `label`, you need to use the attribute `for` with the `input`'s `id` value if it's separate fro the target `input`. More info: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label
Another approach is wrapping the `input` tag with `label`, this way you don't need to manage ids, which need to be unique in the page.

Reply:
Also if I have a <select> or if I have one label with two inputs?
But I'll try with wrapping the `input` tag with the `label`!